humleflue/p6
# feature/pipeline

- First version of c pipeline
- Testing
- Move pipeline to correct dir
- Correcting syntax error
- Fix path
- remove ls
- Make error on purpose
- Make pipeline fail
- Correct syntax error
- a
- b
- c
- Should error
- Fix broken test
- Test new error
- All tests have passed message
- Comments
